### PR TITLE
Bump Inference APIs to Neuron 2.13

### PR DESCRIPTION
**Pull request by JingyaHuang**
- [x] Unblock VAE encoder
- [x] Remove optimized cross-attention score workaround
- [x] Unblock dynamic batching test
- [x] Update documentation

### CODE REVIEW

File: docs/source/guides/models.mdx [hunk 1]
```diff
@@ -201,10 +201,10 @@ with torch.inference_mode():
 Optimum extends 🤗`Diffusers` to support inference on Neuron. To get started, make sure you have installed Diffusers:
 
 ```bash
-pip install diffusers>=0.17.0
+pip install diffusers>=0.20.0
```
Comment (philschmid) [inline docs/source/guides/models.mdx:204]: should we add an optional with [diffusers] to the setup.p?

File: docs/source/guides/export_model.mdx [hunk 2]
```diff
@@ -226,6 +231,7 @@ optimum-cli export neuron --model stabilityai/stable-diffusion-2-1-base \
   --batch_size 1 \
   --height 512 `# height in pixels of generated image, eg. 512, 768` \
   --width 512 `# width in pixels of generated image, eg. 512, 768` \
+  --num_image_per_prompt 4 `# number of images to generate per prompt, defaults to 1` \
```
Comment (philschmid) [inline docs/source/guides/export_model.mdx:234]: is this needed with dynamic batching support now?

**Review by philschmid — APPROVED** [review]
LGTM!

**Review by dacorvo — APPROVED** [review]

**Review by michaelbenayoun — APPROVED** [review]
LGTM

File: docs/source/guides/models.mdx (hunk 1 above)
Comment (JingyaHuang) [reply docs/source/guides/models.mdx]: We already have diffusers as a soft dependency in `setup.py`, let me update the doc using `optimum[neuronx, diffusers]` extra here.

File: docs/source/guides/export_model.mdx (hunk 2 above)
Comment (JingyaHuang) [reply docs/source/guides/export_model.mdx:242]: If we turn on dynamic batching, we can input any batch size / number images per prompt, but by default the dynamic batch size is turned off, so we can keep the snippet this way I think.

**Pull request merged**